### ISSUE TITLE
Only get the latest 100,000 scenarios from db-bot

### ISSUE THREAD

**Issue by noracato** [quintel/etengine#1359]
When downloading and importing a copy from the anonomized database dumps from db-bot you can easily spend up to 10 minutes just waiting around.
This slow expansion of waiting time happened because over the years the engine's database has grown bigger and bigger. As we only use these database dumps for development and help questions from users, we can opt for dumping not the full database, but only the last hunderd thousand scenarios or so (including the special scenarios like II3050). Then the download (currently 3.3GB) and the import will be much quicker.

We can also maintain the current dump and add the _light_ dump as an option to download from db-bot.

What do you think @mabijkerk and @thomas-qah?

**Comment by mabijkerk** [comment]
Good that you identified this issue @noracato. My first preference would go to the _light_ dump option. This however depends on our use of the db-bot dump. If we do not actually need all those scenarios then it does not make sense to download them. Perhaps there are other filters we can apply that would trim the download?

Since we're having a discussion about `keep_compatible` next week anyway, let's include this issue in that meeting.

**Comment by mabijkerk** [comment]
My main question @noracato is what the use is of the dumps from db-bot. Who uses them and why?

**Comment by thomas-qah** [comment]
Hope you don't mind my answering/chiming in: I think the usecases are diverse and numerous. For example, I used the etengine production dump to do the benchmark testing. It was very useful to have them!

I think downloading dump from the production db usually have such a usecase in mind: to inspect how the database is doing in various manners, and also to see how the database is used; how many inputs/sliders do people set on average, how many curves, etc. As a developer it is much easier to inspect and test such things on a local database. You don't want to do that on a live production database because it puts unnecessary extra load on the server, and even more importantly, it can be dangerous.

**Comment by mabijkerk** [comment]
Thanks for the explanation @thomas-qah. In that case I think my preference would be to set the default to 1 month old scenarios, but to allow users (meaning ourselves) to specify a different time limit.

- `ete pro` download default with 1 month old scenarios and all scenarios on `keep_compatible`
- `ete pro 3` download with 3 month old scenarios and all scenarios on `keep_compatible`
- `ete pro 0` download with only scenarios on `keep_compatible` (might be an interesting option to include)

Does this seem feasible for you?

**Comment by noracato** [comment]
Sure. That means we have to save a few different dumps each night (0, 1, 2 and 3) - right @thomas-qah? Which increases the time the server will be very busy and will increase our bill with amazon a bit. 

If we would attract more users from different time zones, it could become a problem in the future. As they would be using the model at the times it will be busy with creating the backups. Not sure how much of a problem it actually is, but just putting it down here!

**Comment by thomas-qah** [comment]
Yes what @noracato writes is correct, but we could of course create a schedule for when each dump gets created. For example:
- Create a full prod dump each month (e.g. a few days before or after the deploy)
- Create a dump with scenarios created during the last 3 months each week
- Create a dump with scenarios created during the last month each day

I think this would decrease the server load significantly, also compared to now :)
